bench-tx/src/utils.rs

```diff
@@ -97,6 +96,27 @@ pub fn get_account_with_default_account_code(
     Account::from_parts(account_id, account_vault, account_storage, account_code, Felt::new(1))
 }
 
+pub fn get_new_pk_and_authenticator(
+) -> (Word, std::rc::Rc<miden_tx::auth::BasicAuthenticator<rand::rngs::StdRng>>) {
+    use std::rc::Rc;
+
+    use miden_objects::accounts::AuthSecretKey;
+    use miden_tx::auth::BasicAuthenticator;
+    use rand::rngs::StdRng;
+    use rand_chacha::{rand_core::SeedableRng, ChaCha20Rng};
```

Comment:
Question: why are the imports here rather than at the top of the file as we usually have?

Reply:
I was just following the style of this function from `testing`, so I just thought that it's some kind of convention. I'll move them to the top of the file.